docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  app:
+    container_name: wst-backoffice
+    image: wst-backoffice
+    build: ./
+    ports:
+    - "8081:8081"
+    depends_on:
+    - postgresqldb
+  postgresqldb:
+    image: postgres
+    volumes:
+    - postgres-data:/var/lib/postgresql/data
+    ports:
+    - "5432:5432"
+    environment:
+    - POSTGRES_PASSWORD=wst!#4pas@*
```

Comment:
Neve ever store your credentials in the code

Reply:
The idea is to create file with variables.
This file should not be in public access.
And here in docker-compose.yml, you can use the variables from that file.
For more information please read this article 
https://www.techrepublic.com/article/how-to-use-docker-env-file/
or this
https://medium.com/better-programming/using-variables-in-docker-compose-265a604c2006

Reply:
done